### ISSUE TITLE
Grows

### ISSUE THREAD

**Issue by connorgunter** [zachkurfirst/trip-planner-server#28]
- Better variable naming
- Gain a better understanding of how the server is connecting to the client

**Comment by zachkurfirst** [comment]
We only used one model (tied client-side auth, locations and activities directly to the trips model). I think for being scalability in future, it would like be preferred to have multiple models and embed the relationships.

**Comment by elliesolhjou** [comment]
I agree, if you add another model like Activity, you would able to perform full CRUD operation and nest tailoring it to each individual trip and it would give you an idea how to plan your trip! for a better user experience you might tie reviews of the activity displayed with each activity so the user has smoother experience to select an activity
